main/procps/template.py

```diff
@@ -8,6 +8,7 @@
     "--disable-modern-top",
     "--without-systemd",
 ]
+configure_gen = []
```

Comment:
usually when dealing with trailing configure_gen, it should be made to regen, unless that is somehow impossible (e.g. cyclic or broken)